ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go

```diff
@@ -346,6 +346,10 @@ func (scbuilder *SiteConfigBuilder) getExtraManifest(dataMap map[string]interfac
 	if err != nil {
 		return nil, err
 	}
+
+	// Manifests to be excluded from merging
+	var excluded_manifests []string
```

Comment:
Perhaps "doNotMerge" would be a better name, given the fact that @pixelsoccupied is working on a different kind of exclusion?

Reply:
Yep so the code I'm working on to filter is run after listing all the default and user provided MCs. So basically on the `dataMap` map and before [L168](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go#L168) 

`doNotMerge` list is something that's provided by the user? Would it be the same thing as a filename under the [exclude](https://github.com/openshift-kni/cnf-features-deploy/pull/1048/files#diff-5a7eeb6977f4204a7d5c1322a7e1fe7957d13801d22c91348770ef8a71c22466R144) list? 

I think the @tliu2021's `func MergeManifests(dataMap map[string]interface{},`'s `dataMap` should be the one after it's run through the filter? [Algorithm](https://docs.google.com/document/d/1xYD-cVlSw6HXUAdWTLad-JailWkaXO6I_u2Fq6VNpyc/edit) here might make more sense

Reply:
Yes, it has been renamed. 

@pixelsoccupied as we discussed, both actions can be done inside 'getExtraManifests' .